VideoWeb/VideoWeb/ClientApp/src/app/services/hearing-venue-flags.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class HearingVenueFlagsService {
+    private _hearingVenueIsScottish = new BehaviorSubject(false);
```

Comment:
Do we want this to assume it isn't Scottish wouldn't a ReplaySubject be better?
```suggestion
    private _hearingVenueIsScottish = new ReplaySubject(1);
```

Reply:
I thought it would be safer to assume scottish crest is false by default as we know exactly where and when it should be set to true (ie when a user actually signs into a particular hearing) 

Would it benefit in using replay subject in anyway or is it wrong to use behavior subject here?

Reply:
Why a ReplaySubject? Is the idea that an undefined value would be falsy as well?

I feel like BehaviorSubject makes sense here.

Reply:
Anything that relies on it will wait for the value to be emitted. Assuming you aren't blocking calls which I didn't see the image wont load. Then when a value is emitted it will. Rather than the wrong crest been shown first.

Reply:
Replay doesn't emit until it has the first value

Reply:
I think we do want it to show HMCTS crest by default though. Do you think the BehaviorSubject could be an issue say if someone refreshed in the waiting room?

Reply:
Yeah that was what I was thinking as it would show the HMCTS to start with even for a SCTS hearing. Granted  it might not be for long but for a slow connection it could be.

Reply:
if you hard reload on the waiting room it will initially be hmcts logo and will get changed when we get conference details when getConference gets called which then updates the logo back to scottish crest

Reply:
you would notice it only if you are really paying attention, however I didnt think that is functionally wrong as by default the app should always show the hmcts crest, and only display the scottish crest based on the hearing venue

Reply:
It's a very good point. When do we set it though because we need a logo to display in other pages before a user has selected a hearing and stuff that is not required to be authorised for.

Reply:
I think the header/footer displaying a different crest on a reload briefly and whether that is acceptable would need to be  a Neil/Lorenzo question.

Reply:
confirmed with Lorenzo that this isn't a show stopper, a seperate tech debt story has been raised to address this:
https://tools.hmcts.net/jira/browse/VIH-8463